<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlacementPage</class>
 <widget class="QWidget" name="PlacementPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QRadioButton" name="carRadio">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Cars</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="carTotal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButtonStartScreen">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemainingLabel_1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Remaining:</string>
   </property>
  </widget>
  <widget class="QLabel" name="indicatorTitleLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Turn Indicator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backButtonStartScreen">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="gridFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="buttonGrid">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="ChooseDirectionLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Choose Direction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="indicatorLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="busTotal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemainingLabel_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Remaining:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="busRadio">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Buses</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bikeRadio">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Bikes</string>
   </property>
  </widget>
  <widget class="QLabel" name="directionDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>If not selected, Vertical is assumed</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemainingLabel_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Remaining:</string>
   </property>
  </widget>
  <widget class="QLabel" name="shipTitleLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Ships:</string>
   </property>
  </widget>
  <widget class="QLabel" name="bikeTotal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeShipLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 75 26pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Ship Placement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="directionCB">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="customRadio">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemainingLabel_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Remaining:</string>
   </property>
  </widget>
  <widget class="QLabel" name="customTotal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>200</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QListWidget" name="shipList">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
